app/assets/v2/js/pages/bounty_details.js

```diff
@@ -419,6 +419,30 @@ var showWarningMessage = function(txid) {
   var interval = setInterval(waitingRoomEntertainment, secondsBetweenQuoteChanges * 1000);
 };
 
+// refresh page if metamask changes
+waitforWeb3(function() {
```

Comment:
Can we make this generic and push it into `shared.js` ?

Reply:
i dont think that all pages require a refresh on network change.. only bounty details